helpers/matrix.yml

```diff
@@ -40,3 +40,4 @@ APM_SERVER_SUITE:
 KUBERNETES_VERSION:
   - '1.14'
   - '1.15'
+  - '1.16'
```

Comment:
I wish we could define the chart versions for upgrade testing here somehow so it is very clear what the test cases are, something like this
```yaml
 - kubernetes_version: 1.14
   upgrade_from_chart_version: 7.0.0-alpha1
 - kubernetes_version: 1.15
   upgrade_from_chart_version: 7.0.0-alpha1
 - kubernetes_version: 1.16
   upgrade_from_chart_version: 7.4.0
```
But I think we can only fit this logic into the jenkins job inline shell script, or the Makefile inline shell script the way the job axes work, so adding into the Makefile makes sense to me.